source/_integrations/emoncms.markdown

```diff
@@ -16,7 +17,17 @@ related:
     title: Configuration file
 ---
 
-The `emoncms` sensor {% term integration %} creates sensors for the feeds available in your local or cloud based version of [Emoncms](https://emoncms.org).
+[Emoncms](https://github.com/emoncms/emoncms) produces a unique identifier starting from version 11.5.7
+
+If you upgrade your existing Emoncms installation to a version shipping a unique identifier, you have to migrate your Emoncms database before restarting Home Assistant:
```

Comment:
I'm kinda against this part, as this the user is going to find out about this too late. We should instead guard for this and prevent users from getting into awkward situations.

../Frenck

Reply:
If the user does nothing or does not migrate the database, the integration will keep working as before. So he should not be in a embarrassing situation.

Reply:
Then I'm not following why this is here?

Reply:
if users do the change, ie migrate emoncms to version 11.5.7 and migrate their database, they will be able to benefit from the unique id. If they forget to migrate the database, the integration will work as before, ie without unique id. Hope this explication is more clear

Reply:
I think we can add it as a tip, but having such a big procedure as the "introduction" to the integration, make not much sense to me?

This has nothing to do with the integration IMHO.

Reply:
Ok understood. I've restored the introduction. Make more sense indeed, sorry. Tell me if it can be ok like that.